docs/gallery/domain/plot_behavior.py

```diff
@@ -0,0 +1,220 @@
+#    as it is acquired, not by trial as is often reshaped for analysis.
+#    Data can be trial-aligned on-the-fly using the trials table. See the :ref:`basic_trials` tutorial
+#    for further information.
+#
+# For position data ``reference_frame`` indicates the zero-position, e.g.
+# the 0,0 point might be the bottom-left corner of an enclosure, as viewed from the tracking camera.
+
+timestamps = np.linspace(0, 50) / 200
+
+position_spatial_series = SpatialSeries(
+    name="SpatialSeries",
+    description="(x,y) position in open field",
+    data=position_data,
+    timestamps=timestamps,
+    reference_frame="(0,0) is bottom left corner",
+)
+
+position_spatial_series
+
+####################
+#
+# .. seealso::
+#    You can learn more about best practices that can be applied to
+#    :py:class:`~pynwb.behavior.SpatialSeries` `here <https://www.nwb.org/best-practices/#timeseries>`_.
+#
+# To help data analysis and visualization tools know that this :py:class:`~pynwb.behavior.SpatialSeries` object
+# represents the position of the subject, store the :py:class:`~pynwb.behavior.SpatialSeries` object inside
+# of a :py:class:`~pynwb.behavior.Position` object, which can hold one or more :py:class:`~pynwb.behavior.SpatialSeries` objects.
+
+position = Position(spatial_series=position_spatial_series)
+
+####################
+# CompassDirection
+# ------------
+#
+# Spatial Series
+# ^^^^^^^^^^^^^^^
+#
+# Analogous to how position can be stored, we can create a :py:class:`~pynwb.behavior.SpatialSeries`
+# object for representing the view angle of the subject.
+#
+# For direction data ``reference_frame`` indicates the zero-axes, for instance
+# “straight-ahead” might be a specific pixel on the monitor, or some other point in space.
+# The unit of measurement for the :py:class:`~pynwb.behavior.SpatialSeries` object should be radians or degrees.
+
+view_angle_data = np.linspace(0, 4, 50)
+
+direction_spatial_series = SpatialSeries(
+    name="SpatialSeries",
+    description="view angle",
+    data=view_angle_data,
+    timestamps=timestamps,
+    reference_frame="bottom left",
+    unit="radians"
+)
+
+direction = CompassDirection(spatial_series=direction_spatial_series,
+                             name='CompassDirection')
+
+####################
+# TODO
+# mention compression of data, timestamps with H5DataIO
+# H5DataIO(timestamps, compression="gzip")
+
+
+####################
+# BehavioralTimeSeries
+# ------------
+#
+# :py:class:`~pynwb.behavior.BehavioralTimeSeries` is an interface for storing continuous behavior data.
+# Create a :py:class:`~pynwb.base.TimeSeries` object that represents the speed/velocity of an animal.
+
+speed_data = np.linspace(0, 0.4, 50)
+
+speed_time_series = TimeSeries(
+    name="speed",
+    data=speed_data,
+    timestamps=timestamps,
+    unit="m/s",
+)
+
+behavioral_time_series = BehavioralTimeSeries(time_series=speed_time_series,
+                                              name='BehavioralTimeSeries')
```

Comment:
```suggestion
behavioral_time_series = BehavioralTimeSeries(
    time_series=speed_time_series,
    name="BehavioralTimeSeries",
    )
```

I recommend [Black formatting](https://black.readthedocs.io/en/stable/)